modules/ROOT/pages/cassandra/cassandra-connector-examples.adoc

```diff
@@ -1,8 +1,11 @@
-= Cassandra - Examples - Mule 4
+= Example for Creating a Cassandra Database Table
```

Comment:
Suggested organization that matches what I've been doing:

H1: Cassandra Examples - Mule 4
  H2:  Create a Cassandra Database Table 
  H2: Manipulate Data in a Cassandra Database